test/Project.toml

```diff
@@ -3,6 +3,8 @@ BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 CUDAKernels = "72cfdca4-0801-4ab0-bf6a-d52aa10adc57"
+CloudMicrophysics = "6a9e3e04-43cd-43ba-94b9-e8782df3c71b"
```

Comment:
Here we don't want to add CloudMicrophysics as a dependency. In our CI we rely on:

https://github.com/CliMA/CloudMicrophysics.jl/blob/3a1b0ab630ffdefb9a83245af29931ff8ae776a6/.github/workflows/ci.yml#L42-L45

Reply:
If you want to test locally without adding the dependency, there is a useful package that handles this: [TestEnv](https://github.com/JuliaTesting/TestEnv.jl)